pkg/kubelet/sysctl/safe_sysctls.go

```diff
@@ -53,27 +49,15 @@ var safeSysctlsIncludeReservedPorts = []string{
 func SafeSysctlAllowlist() []string {
 	if goruntime.GOOS == "linux" {
 		// make sure we're on a new enough kernel that the ip_local_reserved_ports sysctl is namespaced
-		kernelVersion, err := getKernelVersion()
+		kernelVersion, err := utilkernel.GetVersion()
 		if err != nil {
 			klog.ErrorS(err, "Failed to get kernel version, dropping net.ipv4.ip_local_reserved_ports from safe sysctl list")
 			return safeSysctls
 		}
-		if kernelVersion.LessThan(version.MustParseGeneric(ipLocalReservedPortsMinNamespacedKernelVersion)) {
+		if kernelVersion.LessThan(version.MustParseGeneric(utilkernel.IPLocalReservedPortsNamespacedMinSupportedKernelVersion)) {
```

Comment:
Can we merge this PR after https://github.com/kubernetes/kubernetes/pull/118846 merge？

Reply:
sure.